### PR TITLE
Update of README template of the drivers

### PR DESCRIPTION
**Pull request by bzz**
Small style fix + UAST converter dependencies requirement

### CODE REVIEW

**Comment by bzz** [comment]
CI fails with

```
generated bindata is out of sync
make: *** [validate-commit] Error 2
The command "make validate-commit" failed and exited with 2 during .
```

It looks like I need to commit generated resources as well.

**Comment by bzz** [comment]
History re-written

**Comment by bzz** [comment]
Rebased on latest masted and assets re-generated. 

`make validate-commit` passes locally for me :/

**Comment by bzz** [comment]
Rebased on latest master, regenerated binaries, same as before, CI is failing, same as in #87

**Comment by bzz** [comment]
CI passes now. Ping @smola for review

File: etc/skeleton/README.md.tpl [hunk 1]
```diff
@@ -16,12 +16,13 @@ Development Environment
 Requirements:
 - `docker`
 - [`bblfsh-sdk`](https://github.com/bblfsh/sdk) _(go get -u github.com/bblfsh/sdk/...)_
+- UAST converter dependencies _(go get -t -v ./...)_
```
Comment (smola) [inline etc/skeleton/README.md.tpl]: actually this should be covered by `go get -u github.com/bblfsh/sdk/...`? What's the missing dependency?

Comment (bzz) [inline etc/skeleton/README.md.tpl]: That would be awesome, but for now [stretchr/testify](https://github.com/stretchr/testify) is missing so `mvn test` fails \wo it.

**Pull request merged**